src/test/java/de/medizininformatikinitiative/flare/service/StructuredQueryServiceIT.java

```diff
@@ -131,25 +135,23 @@ void setUp() throws URISyntaxException, IOException {
     void execute_Criterion() {
         var query = StructuredQuery.of(CriterionGroup.of(CriterionGroup.of(Criterion.of(ContextualConcept.of(DIAGNOSIS, Concept.of(I08))))));
 
-        var result = service.execute(query).block();
+        var result = getExecutionResult(query);
```

Comment:
Sorry, I did see this in my last review. But we have to test the returned ID's here and not only the size.